src/test/java/TestUtils.kt

```diff
@@ -0,0 +1,8 @@
+import com.intellij.openapi.application.PathManager
+import com.intellij.openapi.util.io.FileUtil
+
+object CodeSyncTestUtils {
+    fun getTestDataPath(): String {
+        return FileUtil.toCanonicalPath("${PathManager.getTempPath()}/")
```

Comment:
How about keeping it inside `tests` directory as `tests/tests_data/` and also add it to .gitignore/.syncignore?

Reply:
I tried doing that, but I could not find a way to get the path to `tests/tests_data/` directory from the code. I will do more search later and change this if necessary.

Reply:
@syedgulahmed has done this as https://github.com/codesyncapp/codesync-intellij/pull/78/files#diff-ab7386c3800fb9121247645db6fa87a597432991a80c364a60cc56fb01a45bcdR6 (PR is still open) Maybe that helps.